x-pack/plugins/ml/public/application/jobs/new_job/common/job_validator/job_validator.ts

```diff
@@ -273,6 +277,14 @@ export class JobValidator {
     this._advancedValidations.categorizationFieldValid.valid = valid;
   }
 
+  public get categorizerMissingPerPartition() {
```

Comment:
these should be checked in `isPickFieldsStepValid` below to disable the `Next` button.

Reply:
Updated in https://github.com/elastic/kibana/pull/75061/commits/38edcdd53460c2fab6fd1d36089c92ad8be8c78f